src/main/java/org/opentripplanner/model/TripIdAndServiceDate.java

```diff
@@ -7,14 +7,6 @@
  * Class to use as key in HashMap containing feed id, trip id and service date
  */
 public record TripIdAndServiceDate(FeedScopedId tripId, ServiceDate serviceDate) {
-  public FeedScopedId getTripId() {
-    return tripId;
-  }
-
-  public ServiceDate getServiceDate() {
-    return serviceDate;
-  }
-
   @Override
   public int hashCode() {
```

Comment:
The `hashCode` and `equals` methods can be also removed. 

See https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/Record.html#equals()